src/ui/components/Scanner/Scanner.tsx

```diff
@@ -62,6 +64,7 @@ const Scanner = forwardRef(
     const [groupId, setGroupId] = useState("");
     const [pastedValue, setPastedValue] = useState("");
     const [scanning, setScanning] = useState(false);
+    const [permission, setPermisson] = useState(true);
```

Comment:
why does it default to `true` and not `false`?

Reply:
Updated